README.md

```diff
@@ -54,6 +54,9 @@ This is quite slow and can take up to one or two hours but can provide good resu
 -   Ruby
 -   Rust
 -   Bash (the binary logical operators -o for "or" and -a for "and" are currently not evaluated due to issues with tree-sitter-bash. For the same reason, the && and || operators are not evaluated if placed after the first heredoc delimiter. Default labels in Switch-statement are treated as regular case labels.)
+-   C
+
+Note: to parse .h files as C instead of C++, use the command line option `parse-h-as-c` or `parse-some-h-as-c`.
```

Comment:
It could be enough to specify only one parameter parse-h-as-c. A list of files can be provided optionally. If there is no list of file paths all header files are counted as c.

Reply:
That would be nice, but it does not seem to work with the yargs library used here. If I change "parse-h-as-c" to type "string" just as the "parse-some-h-as-c" parameter, it does unfortunately still provide the specified default value when using the option while not providing any purther input. So `parse sources -o output.json` is, for the program, indistinguishable from `parse sources -o output.json --parse-h-as-c`

Reply:
Or "all" instead of "true". Of course we cannot exclude a folder named "true" or "all" then and I am not sure if such an option would be intuitive.

Reply:
Hm okay, then let's keep it separated.